src/native_code_wrapper.ts

```diff
@@ -1,27 +1,47 @@
-import { NativeModules, Platform } from 'react-native';
+import { NativeModules } from 'react-native';
 
-const LINKING_ERROR =
-  `The package 'rly-network-mobile-sdk' doesn't seem to be linked. Make sure: \n\n` +
-  Platform.select({ ios: "- You have run 'pod install'\n", default: '' }) +
-  '- You rebuilt the app after installing the package\n' +
-  '- You are not using Expo Go\n';
+const RlyNativeModule =
+  NativeModules.RlyNetworkMobileSdk && NativeModules.RlyNetworkMobileSdk;
 
-const RlyNativeModule = NativeModules.RlyNetworkMobileSdk
```

Comment:
Only question is why the switch away from the LINKING_ERROR?

Reply:
probably can improve this, but in the existing `native_code_wrapper` file I look for `NativeModules.RlyNetworkMobileSdk` and if it's not there I look for the expo module and use that, else "bundle not found" so in theory there shouldn't be a linking error. that said the way you implemented key manager is much cleaner, happy to leave this open and refactor to something like that, but will need to wait until Monday/Tuesday

Reply:
Ahh got it. LGTM.